src/lib/verifier/DAVerifier.sol

```diff
@@ -107,11 +107,9 @@ library DAVerifier {
         uint256 cursor = 0;
         for (uint256 i = 0; i < _sharesProof.shareProofs.length; i++) {
             uint256 sharesUsed = _sharesProof.shareProofs[i].endKey - _sharesProof.shareProofs[i].beginKey;
-            NamespaceNode memory rowRoot =
-                NamespaceNode(_sharesProof.namespace, _sharesProof.namespace, _sharesProof.rowRoots[i].digest);
             if (
                 !NamespaceMerkleTree.verifyMulti(
-                    rowRoot,
+                    _sharesProof.rowRoots[i],
```

Comment:
[note for reviewers]
this is the fix for: https://github.com/celestiaorg/quantum-gravity-bridge/issues/193

Reply:
Can this be extracted into a separate PR?

Reply:
https://github.com/celestiaorg/quantum-gravity-bridge/pull/197 and https://github.com/celestiaorg/quantum-gravity-bridge/pull/160/commits/630e9d8f53a9466eacb6e1c032b4c296192cd82b